pom.xml

```diff
@@ -247,7 +247,7 @@
         <dependency>
            <groupId>org.bukkit</groupId>
            <artifactId>bukkit</artifactId>
-           <version>1.13.2-R0.1-SNAPSHOT</version>
+           <version>1.14-pre5-SNAPSHOT</version>
```

Comment:
Update to 1.14-R0.1-SNAPSHOT

Reply:
I agree, update this to the `R0.1`. While there are unlikely API changes, still best to do.